### ISSUE TITLE
Writing_Linter_Bears.rst: Broken link to diveintopython3.net

### ISSUE THREAD

**Issue by shashank-b** [coala/coala#5944]
The section `Writing the bear` contains a broken link and needs to be updated. The correct link can be easily found by following this issue https://github.com/realpython/python-guide/issues/926.

This could be a nice issue for a newcomer to begin with.

**Comment by tomasn4a** [comment]
On it!

**Comment by C0ffeeMachine** [comment]
please assign me this issue. I think I made the changes rightly :)

**Comment by shashank-b** [comment]
A maintainer would assign the issue to on you soon, @tomasn4a , @C0ffeeMachine

**Comment by tomasn4a** [comment]
Hey @shashank-b , I haven't been assigned to this issue yet, is there anything else I need to do? Thanks a lot.

**Comment by tomasn4a** [comment]
Hey @snowmanstark , not yet, no!

**Comment by snowmanstark** [comment]
Hey @tomasn4a, you started on the issue?

**Comment by tomasn4a** [comment]
@snowmanstark, yeah, done with it, just waiting for it to be assigned to submit the PR

**Comment by snowmanstark** [comment]
Ok, cool.

**Comment by theParadoxicalWizard** [comment]
@li-boxuan can you assign me this issue

**Comment by Dilshaad21** [comment]
@shashank-b since this issue was opened way back and still no one has submitted a PR.Therefore I am going for it.

**Comment by Apoorv01017** [comment]
hello, my request to gitlab as a newcomer has yet not been approved in the meantime I wanted to make myself familiar with coala can someone help me with this issue as I wanted to get started with this.

**Comment by gr455** [comment]
Hey, I am a newcomer and have found the issue (dive into python3 broken website link) and would like to work on the same. It would be greatly appreciated if I am assigned this issue

**Comment by mitchell-frost** [comment]
Is this issue closed? It still shows open so can I work on this?

**Comment by pranshu2610** [comment]
@shashank-b please assign this issue to me

**Comment by Karanjot-singh** [comment]
Is this issue open? I would like to work on this!

**Comment by abhishalya** [comment]
@Karanjot-singh This issue already has a PR. Find another issue and ping me to assign :)

**Comment by Karanjot-singh** [comment]
Sure!
